src/puppetlabs/trapperkeeper/services/webserver/jetty9_core.clj

```diff
@@ -546,6 +546,8 @@
                                maybe-logged)]
       (.setHandler s statistics-handler)
       (.setStopTimeout s (or shutdown-timeout default-graceful-stop-timeout))
+      (when-let [script (:post-config-script options)]
```

Comment:
Yeah, I think best practice would be to do all of the configuration here.  Just wasn't sure if I trust us to always follow that practice.  I'd probably still lean toward putting it in `start-webserver!`, but I understand the argument for leaving it here, too.  I'll defer to your judgement on this.